addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -22,9 +22,9 @@
                         <Percentuale t-esc="format_numbers(abs(line.discount))"/>
                     </ScontoMaggiorazione>
                     <PrezzoTotale t-esc="format_monetary(line_dict['subtotal_price'], currency)"/>
-                    <AliquotaIVA t-if="line.tax_ids.amount_type == 'percent'" t-esc="format_numbers(line.tax_ids.amount)"/>
-                    <AliquotaIVA t-elif="line.tax_ids.amount_type != 'percent'" t-esc="'0.00'"/>
-                    <Natura t-if="line.tax_ids.l10n_it_has_exoneration" t-esc="line.tax_ids.l10n_it_kind_exoneration"/>
+                    <AliquotaIVA t-if="line_taxes.amount_type == 'percent'" t-esc="format_numbers(line_taxes.amount)"/>
```

Comment:
better to have the line_taxes in Python.  It will be easier to inherit for the withholding and we do not need an xpath here.  @lordkrandel